Installer-Linux/linux-hosts-installer.sh

```diff
@@ -5,6 +5,15 @@
 # Copyright - Mitchell Krog - mitchellkrog@gmail.com 
 # https://github.com/mitchellkrogza
 
+#
+# root has to run the script
+#
+if [ `whoami` != "root" ]
+    then
+    printf "You need to be root to do this!\nIf you have SUDO installed, then run this script with sudo \$script.bash\n"
```

Comment:
Please clarify.

The message should be something like:

```
[...] with `sudo ${0}`
```

The reason: `${0}` reports to the way the end-user called the script which means that:
- If I call `script.sh` ==> It will be replaced by `script.sh`
- if I call `/path/to/script.sh` ==> It will be replaced by `/path/to/script.sh`